gnucash/register/register-gnome/gnucash-sheet-private.c

```diff
@@ -418,7 +418,10 @@ draw_cell (GnucashSheet *sheet,
     gtk_render_background (stylectxt, cr, x, y, width, height);
 
     gdk_rgba_parse (&color, "black");
-    gnc_style_context_get_background_color (stylectxt, GTK_STATE_FLAG_NORMAL, &color);
+    if (gtk_style_context_get_state (stylectxt) == GTK_STATE_FLAG_INSENSITIVE)
+        gnc_style_context_get_background_color (stylectxt, GTK_STATE_FLAG_INSENSITIVE, &color);
+    else
+        gnc_style_context_get_background_color (stylectxt, GTK_STATE_FLAG_NORMAL, &color);
```

Comment:
Ah, another case of unnecessary command doubling which can be avoided like this:
```
gnc_style_context_get_background_color (stylectxt,
                      gtk_style_context_get_state (stylectxt),
                      &color);
```